specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2357 @@
+        },
+        "confidence": {
+          "description": "Confidence score.",
+          "$ref": "#/definitions/Confidence"
+        },
+        "elements": {
+          "description": "When includeTextDetails is set to true, a list of references to the text elements constituting this field.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ElementReference"
+          }
+        },
+        "page": {
+          "description": "The 1-based page number in the input document.",
+          "type": "integer",
+          "minimum": 1,
+          "x-nullable": false
+        }
+      }
+    },
+    "FieldValueType": {
+      "type": "string",
+      "description": "Semantic data type of the field value.",
+      "enum": [
+        "string",
+        "date",
+        "time",
+        "phoneNumber",
+        "number",
+        "integer",
+        "array",
+        "object",
+        "selectionMark"
+      ],
+      "x-ms-enum": {
+        "name": "FieldValueType",
+        "modelAsString": false
+      },
+      "x-nullable": false
+    },
+    "Confidence": {
+      "description": "Confidence value.",
+      "type": "number",
+      "minimum": 0,
+      "maximum": 1,
+      "x-nullable": false
+    },
+    "Appearance": {
+      "description": "An object representing the appearance of the text line.",
+      "type": "object",
+      "required": [
+        "style",
+        "styleConfidence"
+      ],
+      "properties": {
+        "style": {
+          "description": "Style of the extracted text: handwriting or printed.",
+          "enum": [
+            "handwriting",
+            "printed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "textStyle",
+            "modelAsString": true
+          },
+          "x-nullable": false
+        },
+        "styleConfidence": {
+          "description": "Confidence value of predicted style.",
+          "$ref": "#/definitions/Confidence"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "Endpoint": {
+      "name": "endpoint",
+      "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
+      "in": "path",
+      "x-ms-skip-url-encoding": true
+    },
+    "FileStream": {
+      "name": "fileStream",
+      "description": ".json, .pdf, .jpg, .png or .tiff type file stream.",
+      "x-ms-parameter-location": "method",
+      "in": "body",
+      "schema": {
+        "$ref": "#/definitions/SourcePath"
+      }
+    },
+    "PageRange": {
+      "name": "PageRange",
+      "in": "query",
+      "description": "Custom page numbers for multi-page documents(PDF/TIFF), input the number of the pages you want to get OCR result. For a range of pages, use a hyphen. Separate each page or range with a comma or space.",
```

Comment:
Let's not say "or space" since the Swagger does not indicate this.  By default, the delimiter is ",".

Reply:
done.